attestation-agent/attester/src/se/ibmse.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) Copyright IBM Corp. 2024
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use anyhow::{anyhow, Result};
+use log::debug;
+use pv::{
+    request::BootHdrTags,
+    uv::{AttestationCmd, ConfigUid, UvDevice},
+};
+use serde::{Deserialize, Serialize};
+use serde_json;
+use serde_with::{base64::Base64, serde_as};
+
+#[allow(unreachable_code)]
+pub fn is_se_guest() -> bool {
+    #[cfg(not(target_arch = "s390x"))]
+    return false;
```

Comment:
As we already mark the code path here would only be executed when `target_arch` is `s390x` [here](https://github.com/confidential-containers/guest-components/pull/492/files#diff-8fd997b18e87172dbdcec4c6ab0268c8d5b8e4dcd7b3a4d493ab1f9519d6c10aR137-R138), we might not need this condition.